src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  gap: 1px;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  width: 7 * ($cell-size + 1px) + 2 * $cell-padding;
```

Comment:
add more padding to the top, read checklist

Reply:
Could you tell me which paddings are we talking about? I set 10px paddings as it has been told in checklist
<img width="818" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/110922653/0d932a8c-de11-4073-96c7-856633ed62fb">